src/openforms/js/components/admin/form_design/registrations/objectsapi/fields/DocumentTypes.js

```diff
@@ -0,0 +1,28 @@
+import {FormattedMessage} from 'react-intl';
+
+import Fieldset from 'components/admin/forms/Fieldset';
+
+import CatalogueSelect from './CatalogueSelect';
+
+export const DocumentTypesFieldet = () => (
+  <Fieldset
+    title={
+      <FormattedMessage
+        description="Objects registration: document types"
+        defaultMessage="Document types"
+      />
+    }
+    collapsible
+    fieldNames={['catalogue']}
+  >
+    <div className="description">
+      <FormattedMessage
+        description="New document types informative message"
+        defaultMessage={`The legacy document types configuration will be ignored
+        as soon as a catalogue is selected, even if you don't select any document
+        type in the dropdowns.`}
```

Comment:
I might be misunderstanding this: in #4577 it's mentioned that we're replacing documenttype URLs with `(catalogus, description)`, but where do I select the description?

Also, since this does deprecate the other documenttype fields, I think it might be good to explain here what this select is needed for

Reply:
as mentioned in standup, #4589 is implementing that aspect :) I wanted to use this PR to properly set up react-select and have one concrete usage for it, so that the rest of the feature/improvement can be implemented with less distractions.

Reply:
that sounds good :+1: